spec/intl.html

```diff
@@ -1466,7 +1732,7 @@ <h1>Temporal.Calendar.prototype.era ( _temporalDateLike_ )</h1>
               1. Set _temporalDateLike_ to ? ToTemporalDate(_temporalDateLike_).
             1. If _calendar_.[[Identifier]] is *"iso8601"*, then
               1. Return *undefined*.
-            1. Let _era_ be the result of implementation-defined processing of _temporalDateLike_ and _calendar_.[[Identifier]].
+            1. Let _era_ be ! CalendarDateEra(_calendar_.[[Identifier]], _temporalDateLike_).
             1. Return _era_.
```

Comment:
```suggestion
            1. Return ! CalendarDateEra(_calendar_.[[Identifier]], _temporalDateLike_).
```
(the name of the variable no longer serves any self-documentation purpose)

Reply:
Thanks. done!